docs/reference/docs/refresh.asciidoc

```diff
@@ -20,7 +20,7 @@ replying. This doesn't force an immediate refresh, rather, it waits for a
 refresh to happen. Elasticsearch automatically refreshes shards that have changed
 every `index.refresh_interval` which defaults to one second. That setting is
 <<dynamic-index-settings,dynamic>>. The <<indices-refresh>> API will also
-cause the request to return, as will setting `refresh` to `true` on any of the
+cause an immediate refresh, as will setting `refresh` to `true` on any of the
```

Comment:
I intended the last sentence to be about what things cause the request that has `refresh=wait_for` to return. The refresh API will do it, as will setting `refresh=true` on one of the write requests that hits the shard that that `refresh=wait_for` request is waiting on.

Maybe it'd make sense to rewrite the last sentence to something like: "The <<indices-refresh>> API will also cause a refresh, in turn causing requests with `refresh=wait_for` to return. Setting `refresh=true` on requests that support it will also cause a refresh, also causing requests with `refresh=wait_for` to return."

Or something. I agree that what is in the docs now is unclear but I'm not sure your change makes it a ton better.